packages/gatsby-theme-carbon/src/components/LeftNav/LeftNavItem.js

```diff
@@ -2,7 +2,7 @@ import React, { useContext } from 'react';
 import { Link } from 'gatsby';
 import { Location } from '@reach/router';
 import cx from 'classnames';
-import useNetwork from 'react-use/lib/useNetwork';
+import {useNetworkState} from 'react-use';
```

Comment:
A bit nit picky here, but make sure to apply formatting (i.e. spaces round this)

Reply:
Applied lint fix.